collectors/charts.d.plugin/README.md

```diff
@@ -25,12 +16,20 @@ The scripts should have the filename suffix: `.chart.sh`.
 
 ## Configuration
 
-`charts.d.plugin` itself can be configured using the configuration file `/etc/netdata/charts.d.conf`
-(to edit it on your system run `/etc/netdata/edit-config charts.d.conf`). This file is also a BASH script.
+> ### Info
```

Comment:
I addressed this in the latest commit, but as a user I would like to have a boilerplate message when I see configuration in a document for the first time, because I might not know about edit config, a link to the configuration document doesn't hint that I have to use a script, so if I am not going by the book I will end up searching and opening `.conf` files